ci/pipelines/meta/deploy_meta.sh

```diff
@@ -35,7 +35,21 @@ for cmd in Jinja2 PyYAML; do
 done
 
 if [ -z $(command -v gcloud) ]; then
-  echo "Install gcloud"
+  echo "Install gcloud (try brew cask install google-cloud-sdk, or see https://cloud.google.com/sdk/docs/downloads-interactive#mac)"
+  exit 1
+fi
+
+forks=$(git remote -v | sed -e 's/.*github.com.//' -e 's#/.*##' | sort -u)
```

Comment:
This part is not really useable at this point with incoming changes to deploy_meta.sh, we're shifting to a properties-style file of bash environment variables with local overrides. In that world, you create "meta.properties.local" and set GEODE_FORK to your upstream once and then you don't have to worry about it after that.